src/utils/poolInfo.ts

```diff
@@ -40,7 +41,9 @@ const isUniPool = (address: string, chainId: ChainId | undefined) => {
 }
 
 const isSushiPool = (address: string, chainId: ChainId | undefined) => {
-  if (chainId === ChainId.KOVAN) {
+  if (chainId === ChainId.MATIC) {
+    return SUSHI_POOLS_ADDRESSES_MATIC.includes(address.toLocaleLowerCase())
+  } else if (chainId === ChainId.KOVAN) {
```

Comment:
Might be nicer just to do...
```suggestion
  }
  
  if (chainId === ChainId.KOVAN) {
```

Reply:
Using else if will let the compiler skip the other conditions if one of the conditions on top is true. This is the way I read else if statements as well so personally I would prefer else if rather than multiple if statements.